src/Authentication/ClientCertificate/Type/GetServiceTicketResponse.php

```diff
@@ -7,27 +7,30 @@
 class GetServiceTicketResponse implements ResultInterface
 {
     /**
+     * Service ticket for the specified service.
+     *
      * @var string
      */
-    private $serviceTicket;
+    private string $serviceTicket;
 
     /**
      * @return string
      */
-    public function getServiceTicket()
+    public function getServiceTicket() : string
     {
         return $this->serviceTicket;
     }
 
     /**
      * @param string $serviceTicket
-     * @return GetServiceTicketResponse
+     * @return static
      */
-    public function withServiceTicket($serviceTicket)
+    public function withServiceTicket(string $serviceTicket) : static
     {
         $new = clone $this;
         $new->serviceTicket = $serviceTicket;
 
         return $new;
     }
 }
+
```

Comment:
double newline

Reply:
This code is generated by the library, check the readme [here](https://github.com/openeuropa/epoetry-client/blob/2.x/README.md#generate-soap-related-classes).